### PR TITLE
Update Resources to match client

### PR DESCRIPTION
**Pull request by BrunoMilitoFrancois**

### CODE REVIEW

File: ZPastel.Service/Repositories/IPastelRepository.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 using System.Threading.Tasks;
 using ZPastel.Model;
 
-namespace ZPastel.Persistence.Contract
+namespace ZPastel.Service.Repositories
```
Comment (joseluz) [inline ZPastel.Service/Repositories/IPastelRepository.cs]: Create a new layer Core and moved the interfaces to this layer

File: ZPastel.API/Converters/PastelConverter.cs [hunk 2]
```diff
@@ -10,10 +10,13 @@ public PastelResource ConvertToResource(Pastel pastel)
             return new PastelResource
             {
                 Id = pastel.Id,
-                Ingdredients = pastel.Ingredients,
```
Comment (joseluz) [inline ZPastel.API/Converters/PastelConverter.cs:13]: test

Comment (BrunoMilitoFrancois) [reply ZPastel.API/Converters/PastelConverter.cs:13]: works!

File: ZPastel.Service/Repositories/IPastelRepository.cs (hunk 1 above)
Comment (BrunoMilitoFrancois) [reply ZPastel.Service/Repositories/IPastelRepository.cs]: Fixed

File: ZPastel.API/Resources/OrderResource.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+namespace ZPastel.API.Resources
+{
+    public class OrderResource
+    {
+        public long Id { get; set; }
+        public decimal TotalPrice { get; set; }
+        public IList<OrderItemResource> OrderItems { get; set; } = new List<OrderItemResource>();
+        public string CreatedByUsername { get; set; }
+        public long CreatedById { get; set; }
+        public DateTime CreatedOn { get; set; }
```
Comment (joseluz) [inline ZPastel.API/Resources/OrderResource.cs]: Can this be nullable?

Comment (BrunoMilitoFrancois) [reply ZPastel.API/Resources/OrderResource.cs]: Changed CreatedBy and LastModifiedBy from OrderResource, OrderItemResource and PastelResource to be nullable

**Pull request merged**